features/support/ui/your_details.rb

```diff
@@ -12,6 +12,8 @@ class YourDetailsPage < SitePrism::Page
     element :minimum_contribution_button, '#your_details_form_contribution_preference_minimum'
     element :full_contribution_button, '#your_details_form_contribution_preference_full'
 
+    element :salary_below_threshold_callout, '[data-dough-callout-radio-disabled]'
```

Comment:
@margOnline is this guaranteed to be a unique identifier for this field, `data-dough-callout-radio-disabled`? I know this tool doesn't have other radio fields so this will stand unique only for as a long as there's no radio button with a dough callout added to it. 

I'm happy for this to go out but just raising the awareness.